scripts/docker_test_ml.sh

```diff
@@ -80,7 +81,7 @@ test_pytorch()
 		if [ ! -d "test/data" ]; then
 			mkdir test/data
 		fi
-		wget --quiet --show-progress --progress=bar:force:noscroll --no-check-certificate $DATA_URL -O test/data/$DATA_NAME.tar.gz
+		wget --quiet --no-check-certificate $DATA_URL -O test/data/$DATA_NAME.tar.gz
```

Comment:
@atharvanan1 should we set the additional parameters as a default and override if set via environment variable or similar?

Reply:
This should be based on ID field from `/etc/os-release` now. See - https://github.com/atharvanan1/jetson-containers/blob/aa35f7f172992faea45ebacac70aacc6df2db31d/scripts/l4t_version.sh#L26